lib/templates/notificationTemplateNgsi2.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Just to confirm... this is a JSON Schema template?

Reply:
Yes, similar to this one for NGSIv1: https://github.com/dcalvoalonso/iotagent-node-lib/blob/task/activeAttsNgsi2/lib/templates/notificationTemplate.json

They are the input for [this library](https://github.com/flatiron/revalidator) that allows validating JSON schemas.

You can see how it is used in https://github.com/dcalvoalonso/iotagent-node-lib/blob/task/activeAttsNgsi2/lib/services/northBound/contextServer.js#L750

Reply:
NTC